modules/GenericPowermeter/serialization.hpp

```diff
@@ -0,0 +1,178 @@
+// SPDX-License-Identifier: Apache-2.0
```

Comment:
I see that this code is tested, but where is it used?

Reply:
Its (for now) used in https://github.com/EVerest/everest-core/pull/443/files

Reply:
Ok, fine by me.